minicluster/src/main/java/org/apache/accumulo/minicluster/MiniAccumuloRunner.java

```diff
@@ -93,6 +94,7 @@ private static void printProperties() {
     System.out.println("#" + TSERVER_MEMORY_PROP + "=128M");
     System.out.println("#" + ZOO_KEEPER_MEMORY_PROP + "=128M");
     System.out.println("#" + JDWP_ENABLED_PROP + "=false");
+    System.out.println("#" + EXISTING_ZOO_KEEPERS_PROP + "=localhost:2181");
```

Comment:
This is to print a text file with the default values that can then be altered as needed.